java/compaction/compaction-job-execution/src/test/java/sleeper/compaction/jobexecution/CompactSortedFilesIT.java

```diff
@@ -191,6 +154,49 @@ void filesShouldMergeCorrectlyAndStateStoreUpdatedByteArrayKey() throws Exceptio
         // - Check DynamoDBStateStore has correct active files
         assertThat(stateStore.getActiveFiles())
                 .usingRecursiveFieldByFieldElementComparatorIgnoringFields("lastStateStoreUpdateTime")
-                .containsExactly(dataHelper.expectedLeafFile(compactionJob.getOutputFile(), 200L, new byte[]{0, 0}, new byte[]{1, 71}));
+                .containsExactly(dataHelper.expectedRootFile(compactionJob.getOutputFile(), 200L));
+    }
+
+    @Nested
+    @DisplayName("Generate test data")
+    class GenerateTestData {
```

Comment:
I think for this to read well these should probably be next to the test the data is for. Maybe a nested class for dealing with a string key and another for dealing with a byte array key?